orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/gce/GoogleServerGroupCreator.groovy

```diff
@@ -49,34 +50,39 @@ class GoogleServerGroupCreator implements ServerGroupCreator, DeploymentDetailsA
       operation.credentials = operation.account
     }
 
-    operation.image = operation.image ?: getImage(stage)
-
-    if (!operation.image) {
-      throw new IllegalStateException("No image could be found in ${stage.context.region}.")
+    if (stage.context.imageSource == "artifact") {
+      operation.imageSource = "ARTIFACT"
```

Comment:
This upper/lowercase conversion is a little odd

Reply:
Makes sense, I'll change this

Reply:
Agreed---ideally we'd set upper-case constants in the pipeline definition in deck, but unfortunately deck sets these as lower-case.  Given that we can't change that to upper-case without breaking existing pipelines or writing a pipeline migrator, for now leaving this as is.